src/main/kotlin/step3/CarRace.kt

```diff
@@ -0,0 +1,23 @@
+package step3
+
+import step3.view.DisplayView
+import step3.view.InputView
+
+class CarRace {
+    private val carList: List<Car>
+    private val carMover: CarMover = CarMover()
+    private val numOfCar: Int = InputView.inputNumOfCar()
+    private val numOfMove: Int = InputView.inputNumOfGame()
+
+    init {
+        carList = List(numOfCar) { Car() }
+    }
```

Comment:
이 부분은 순서를 바꾸면 인라인으로 작성해 볼 수 있겠네요 :)
```suggestion
    private val carMover: CarMover = CarMover()
    private val numOfCar: Int = InputView.inputNumOfCar()
    private val numOfMove: Int = InputView.inputNumOfGame()
    private val carList: List<Car> = List(numOfCar) { Car() }
```